src/components/UTIL_NavigateBack.component

```diff
@@ -0,0 +1,28 @@
+<apex:component >
+    <apex:attribute name="recordId"
+        description="The recordId to navigate to."
+        type="Id"
+        required="true"
+        />
+    <apex:attribute name="redirect"
+        description="Prevent redirect unless set to true."
+        type="Boolean"
+        default="false"
+        />
+
+    <script type="text/javascript">
+        var redirect = {!redirect};
+        if (redirect) {
+            if ((typeof sforce != 'undefined') && sforce && (!!sforce.one)) {
+                // Manage navigation in Lightning Experience & Salesforce1
+                    sforce.one.back(true);
```

Comment:
sforce.one.back() takes a refresh parameter, which I set to true above which is what forces the page to refresh. There doesn't seem to be an equivalent refresh for any other method.